hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -546,11 +568,6 @@ public Builder compactionSmallFileSize(long smallFileLimitBytes) {
       return this;
     }
 
-    public Builder compactionRecordSizeEstimateThreshold(double threshold) {
```

Comment:
trying to understand why these are removed. There may be clients using this outside this repo

Reply:
my bad, there was no usage for this method, hence thought of removing. Let me revert that.